### ISSUE TITLE
Naming Squads + Possible Solo-Squad Solution

### ISSUE THREAD

**Issue by Snap-2** [UncreatedStaff/UncreatedWarfare#252]
According to users it was a feature prior. Feature of which was removed from the game, supposedly although not specified for the reason of abused names.

This _could_ bring more people to use squads with their friends and make grouping up with people more fun.

Also another thing that would be interesting to see added is a roughly 1-2 minute disband timer _(after staging/preparation phase)_ for single man squads. This is an annoying occurrence of the server plenty of people have complained about. This could be accompanied by a small cooldown timer of 5-6 minutes on making a new squad, albeit not necessary.

**Possible downsides:**

- People abusing name system with derrogatory, possibly rule breaking words AGAIN.
- Making it harder for moderation (for a non game affecting change) due to what was mentioned prior.
- People malding because their squad got disbanded mid-game.
- It's not much so of a necessary change but more so a small quality of life fun update a few people would probably like to see added back.

**Comment by Lusienum** [comment]
Agree, this would be very nice to see this feature on our server again
